.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "codegen/botocore"]
```

Comment:
Good call. I actually used the latest tag, 1.3.7, to lock rusoto to a specific release rather than whatever commit happened to be HEAD between releases. When rusoto needs to update botocore, just changing the tag should suffice.